src/models/target/index.ts

```diff
@@ -32,20 +32,20 @@ export interface ITarget {
   title: string;
   shortCode: string;
   description: string;
-  standards: IStandards[];
+  standards?: IStandards[];
```

Comment:
We should use union types `standards: IStandards | undefined` for this and use `undefined` specifically. This says "the property has not been initialized" vs "the property is not available"

Reply:
The inventor of `null` in javascript also says that you should use `undefined`